data/MahekUnnisa.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Mahek Unnisa",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Textile technologist to Self-taught Web developer, Technical Blogger, Open Source, and DevOps enthusiast. I talk about Python, Javascript, and my learnings.",
+  "avatar": "https://github.com/MahekUnnisa.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/TheMahekUnnisa",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Linkedin",
+      "url": "https://linkedin.com/in/mahekunnisa",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/MahekUnnisa",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https:mahekunnisa.hashnode.dev",
+      "icon": "FaHashnode"
```

Comment:
use `SiHashnode` this for the Hashnode icon

Reply:
Thanks for letting me know @Aadarsh805 . I have changed the file accordingly.